test/unit/openapi/streaming/subscription-spec.js

```diff
@@ -387,7 +387,10 @@ describe("openapi StreamingSubscription", () => {
 			});
 		});
 
-		it ("ignores a unsubscribe followed by a subscribe when waiting for an action to respond", (done) => {
+		/**
+		 * Unsubscribe before subscribe is required for modify action.
+ 		 */
+		it ("accept unsubscribe followed by a subscribe when waiting for an action to respond", (done) => {
```

Comment:
I'd rather have an explicit modify action and ignore subscribe followed by unsubscribe..